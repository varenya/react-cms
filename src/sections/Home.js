import React, {Component} from 'react';

//MUICSS Components
//import {Row, Col} from './GridComponent';

//Project Components
//import Header from './Header';
//import ComponentsMenu from './ComponentMenu';

//MUI CSS file
import 'flexboxgrid/css/flexboxgrid.css';

import ComponentMenu from './ComponentMenu';
import Header from '../components/Header'

import './Home.css'

class Home extends Component {

    render() {
        return (
          <div>
            <Header welcome="REACT CMS"  />
            <div className="pure-g home">
              <div className="pure-u-1-3">
                <ComponentMenu />
              </div>
              <div className="pure-u-2-3">World</div>
            </div>
          </div>
        );
    }

}

export default Home;
