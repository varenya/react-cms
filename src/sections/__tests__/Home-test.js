import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

test('Check <Home /> is getting rendered properly without errors!' , () => {
  const component = renderer.create(
    <Home  />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
