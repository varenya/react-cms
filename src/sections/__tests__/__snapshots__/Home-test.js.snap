exports[`test Check <Home /> is getting rendered properly without errors! 1`] = `
<div>
  <div
    className="header">
    <div
      className="home-menu pure-menu pure-menu-horizontal pure-menu-fixed">
      <a
        className="pure-menu-heading"
        href="#">
        REACT CMS
      </a>
    </div>
  </div>
  <div
    className="pure-g home">
    <div
      className="pure-u-1-3">
      <div
        className="sidebar"
        id="component-menu">
        <div
          className="pure-menu">
          <div
            className="pure-menu-heading">
            Components
          </div>
          <ul
            className="pure-menu-list">
            <li
              className="pure-menu-item"
              style={
                Object {
                  "color": "#fff",
                }
              }>
              <a
                className="pure-menu-link"
                href="#">
                Test
              </a>
            </li>
            <li
              className="pure-menu-item"
              style={
                Object {
                  "color": "#fff",
                }
              }>
              <a
                className="pure-menu-link"
                href="#">
                Test
              </a>
            </li>
            <li
              className="pure-menu-item"
              style={
                Object {
                  "color": "#fff",
                }
              }>
              <a
                className="pure-menu-link"
                href="#">
                Test
              </a>
            </li>
            <li
              className="pure-menu-item"
              style={
                Object {
                  "color": "#fff",
                }
              }>
              <a
                className="pure-menu-link"
                href="#">
                Test
              </a>
            </li>
            <li
              className="pure-menu-item"
              style={
                Object {
                  "color": "#fff",
                }
              }>
              <a
                className="pure-menu-link"
                href="#">
                Test
              </a>
            </li>
            <li
              className="pure-menu-item"
              style={
                Object {
                  "color": "#fff",
                }
              }>
              <a
                className="pure-menu-link"
                href="#">
                Test
              </a>
            </li>
            <li
              className="pure-menu-item"
              style={
                Object {
                  "color": "#fff",
                }
              }>
              <a
                className="pure-menu-link"
                href="#">
                Test
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="pure-u-2-3">
      World
    </div>
  </div>
</div>
`;
